CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 4.7.2 Whiskey Sour (2023-02-08)
```

Comment:
I think some entries are missing, maybe the issues from "prior releases" are being left out by tool_belt? I see at least the orphan deletion issue isn't in here.

Reply:
@ianballou added the 5 other issues, let me know how this looks.